locales/en-US/app.ftl

```diff
@@ -144,6 +144,7 @@ enableExperimentTransition = Enabling...
 measurements = Your privacy
 experimentPrivacyNotice = You can learn more about the data collection for {$title} here.
 contributorsHeading = Brought to you by
+contributorsExtraURL = Learn more
```

Comment:
Could we name this something more descriptive for localizers? `contributorsExtraLearnMore` maybe?